.github/scripts/get-previous-tag.js

```diff
@@ -0,0 +1,22 @@
+
+function getPreviousTag(version, allReleases) {
```

Comment:
(picking random line)
Is this issue related to what I got on another repo (cloudserver), where the action would generate the release notes from the last released tag **of the release tagged as "latest**? Typically, if we release last the 2.6.X, then a 2.9.X, the latter will have all the changelogs since the 2.6.X if it's marked as the "latest".

So my question is: if this is indeed the same problem, it affects all repos, and eventually we can have the same error again, where the release fails as the release notes are too long (e.g., a 7.10.X release in cloudserver, and then a 8.8.X).

Reply:
In such case, shouldn't we use a scality action based on the current one, so it benefits all repos?

Reply:
yes, it could affect all repos, though not sure as it does not happen all the time or on every repo: and (even on repo) it sometimes works as expected.... maybe a combination of where the release is made, what the default branch is and on which branch the "latest" release is (like having the latest release on the default branch or not).

let's proceed incrementally, take the fix here first, and see how we generalize (we may want to generalize more than just the release notes) as it is a separate increment.